liquibase-core/src/test/groovy/liquibase/sqlgenerator/core/CreateProcedureGeneratorTest.groovy

```diff
@@ -42,4 +48,44 @@ class CreateProcedureGeneratorTest extends Specification {
         "some txt; \n-- line cmt \n--another \n /* and block\\/ */\\t\n\ndefine something;\n/\t--last comment\n\n\n\t--another\n/*****another\nblock\n***/" | "/"       | "some txt; \n-- line cmt \n--another \n /* and block\\/ */\\t\n\ndefine something;\n"
 
     }
+
+    def "correct SQL is generated for a procedure"() {
```

Comment:
Since you are testing addSchemaToText, the test name should start with "addSchemaToText" so we know what is being checked.

Since this method and the one below are variations on the same test, it would be best to merge them into a single test with different "when" values. The table structure gets a bit tough with a long string like that but see if you can get it to work. Otherwise, there is the "<<" style with variables.